eng/build.yml

```diff
@@ -172,35 +172,36 @@ jobs:
               HelixAccessToken: $(_HelixApiToken)
               RunAoTTests: 'false'
 
-      - task: PublishTestResults@2
-        displayName: Publish Test Results	
-        inputs:	
-          testResultsFormat: xUnit	
-          testResultsFiles: 'artifacts/TestResults/$(_BuildConfig)/*.xml'	
-          testRunTitle: '$(_AgentOSName)_$(Agent.JobName)'	
-          buildPlatform: '$(BuildPlatform)'	
-          buildConfiguration: '$(_BuildConfig)'	
-        condition: always()
+      - ${{ if eq(variables['_HelixOnly'], false) }}:
```

Comment:
I've added this variable `_HelixOnly` to denote if the particular job only has Helix tests. When this is true, there aren't test results to publish using this task. Without this, we get warnings about there being no test results available to publish.